source/develop/release-management/features/network/provider-physical-network.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Provider Physical Network
+category: feature
+authors: phoracek,amusil
+feature_name: Provider Physical Network
+feature_modules: engine,vdsm,ovn-provider
+feature_status: In Development
+---
+
+# Provider Physical Network
+
+## Summary
+
+This feature extends [ovirt-ovn-provider](https://www.ovirt.org/develop/release-management/features/network/ovirt-ovn-provider/) supporting physical network access. With this feature, user will be able connect external OVN network to a physical (oVirt native) network.
+
+Progress of the feature is tracked on [ovn-localnet Trello](https://trello.com/b/lxIBEn0A/ovn-localnet). Related patches could be found on [gerrit topic:localnet](https://gerrit.ovirt.org/#/q/topic:localnet).
+
+### Owner
+
+- Feature Owner: Petr Horáček (phoracek)
+- E-mail: <phoracek@redhat.com>
+
+### Benefit to oVirt
+
+With this feature existing OVS VM networking could be replaced with OVN, providing traditional oVirt networking. That would also allow us to drop a part of VM live migration hook.
+
+Physical network access will be required once NAT and routing are introduced OVN provider.
+
+## Usage
+
+### Set cluster network switch type to OVS
+
+This feature requires cluster switch type set to OVS. You can enable OVS networking with following steps.
+
+1) **Enable OVS hook for live migration on all vdsm hosts.** Add `migration_ovs_hook_enabled = true` to `[vars]` section in `/etc/vdsm/vdsm.conf`. And restart vdsm with `systemctl restart vdsmd supervdsmd`.
+
+1) **Set Cluster switch type to OVS.** Open `Edit` on selected cluster and set `Switch Type` to `OVS (experimental)`.
+
+1) **Set OVS networking on all vdsm hosts.** Go over each Host, put it to `Maintenance`, `Sync All Networks` and `Activate`.
+
+### Create an external network on top of a physical network
+
+There are several ways how to use this feature.
+
+- **Select physical network via custom values.** When creating a new external network, fill physical network name to `Physical Network`. If the physical network has VLAN tagging enabled, also check `Enable VLAN tagging` and copy physical network's tag there. Please note, that physical network name references to VDSM network name, which is not necessarily the same as oVirt network name (in case it has more than 15 characters or contains special characters).
```

Comment:
"via custom values"
This sounds as if there were multiple custom values, like for DNS's.
maybe change this to:
-> via a custom physical network name
or
-> via a custom value